### ISSUE TITLE
embedded chat/community

### ISSUE THREAD

**Issue by sonoftherock** [javascript-tutorial/en.javascript.info#1149]
I have been working on a chat widget for online tutorials and would love it to be embedded on javascript.info. I think it would be a good replacement to disqus and am willing to do it for completely free given that this is an open-source, non profit endeavor. Please check out the [demo](https://bloomchat.app/tutorials/demo) and [landing page](https://bloomchat.app/) of what I've built.

**Comment by iliakan** [comment]
Should we make a Telegram channel? Or a Slack channel? Or something else for the community?

**Comment by sonoftherock** [comment]
Hey Ilya, I think Bloom would work great on javascript.info. You can think of it as Slack that can be embedded onto the website so that people can chat directly from the website, instead of having to go back and forth between javascript.info and a slack group. We are actually soon gonna be supporting github login and, if you are willing, we can customize authentication to pull from the account information on javascript.info so that people on the website will not even have to make a separate account, whereas they would have to on Telegram or Slack.

**Comment by iliakan** [comment]
There are so many options. I've never heard of Bloom.

Let's hear what people say. And also, if they are interested, because there are so many communities around.

**Comment by sonoftherock** [comment]
Yes, sounds good. I am an undergraduate student at Yale who built this platform because I often use online resources myself. Bloom is made primarily for online courses with an instructor, but we are interested in seeing how our tool could be useful for self studying with resources like JavaScript.info as well. If you want to give Bloom a chance, just let me know.

**Comment by Violet-Bora-Lee** [comment]
Hi, I'm also considering a chat system for Korean contributors. I think there are well-maintained communication systems for git repository such as Gitter, Slack.

I was trying to ask @iliakan for creating Gitter community here but I saw this issue. So, I leave my opinion here.

**Comment by iliakan** [comment]
@Violet-Bora-Lee why do you prefer gitter over other means? What's popular in Korea?

**Comment by Violet-Bora-Lee** [comment]
Korean contributors actually are using Kakao chat application. It's the most famous chat app among Korean.
I don't prefer anything but I want to change communication means nowadays to log any activities.

Korean repository is growing fastly because I've introduced this project on some conferences and seminars. So, I'm thinking of how to improve the translation process.

**Comment by iliakan** [comment]
I created the gitter room and invited you there. 

When the 1st part is ready we can publish the Korean version.

**Comment by Violet-Bora-Lee** [comment]
Awesome! Thanks, Ilya 🥇 🥇 🥇
I'll notify Korean contributors we can use new Gitter room. :)

**Comment by iliakan** [comment]
@Violet-Bora-Lee by the way, there's bot to use for translation, it adds/removes labels and requests reviews for PRs, labels translations in the issue.

https://javascript.info/translate/bot.

**Comment by Violet-Bora-Lee** [comment]
Yeah. I think the auto-labeling feature is cool 💯
